src/document/DocumentCommandHandlers.js

```diff
@@ -460,7 +472,7 @@ define(function (require, exports, module) {
      * fullPath: is in the form "path[:lineNumber[:columnNumber]]"
      * lineNumber and columnNumber are 1-origin: lines and columns are 1-based
      */
-
+    
```

Comment:
Trailing whitespace creating unnecessary diff. There are other cases like this in your commit. I suggest to review them for your next submission.